src/ocm/components/clusterDetail/ClusterProperties.tsx

```diff
@@ -74,82 +75,112 @@ const getDiskEncryptionEnabledOnStatus = (diskEncryption: DiskEncryption['enable
+            title="Network management type"
+            value={getManagementType(cluster.userManagedNetworking)}
+            testId="network-management-type"
+          />
+          <DetailItem
+            title="Networking Type"
+            value={getNetworkType(cluster.networkType)}
+            testId="networking-type"
+          />
+        </DetailList>
       </GridItem>
-    )}
-    <GridItem md={6} data-testid="cluster-details">
-      <DetailList>
-        {externalMode ? undefined : (
+      <GridItem md={6}>
+        <DetailList>
           <DetailItem
-            title="OpenShift version"
-            value={cluster.openshiftVersion}
-            testId="openshift-version"
+            title="Cluster network CIDR (IPv4)"
+            value={selectClusterNetworkCIDR(cluster)}
+            testId="cluster-network-cidr-ipv4"
           />
-        )}
-
-        <DetailItem title="Base domain" value={cluster.baseDnsDomain} testId="base-dns-domain" />
-        <DetailItem
-          title={getCpuArchTitle()}
-          value={cluster.cpuArchitecture}
-          testId="cpu-architecture"
-        />
-        <DetailItem
-          title="API IP"
-          value={cluster.apiVip}
-          isHidden={!cluster.apiVip}
-          testId="api-vip"
-        />
-        <DetailItem
-          title="Ingress IP"
-          value={cluster.ingressVip}
-          isHidden={!cluster.ingressVip}
-          testId="ingress-vip"
-        />
-        <DetailItem
-          title="Network management type"
-          value={getManagementType(cluster.userManagedNetworking)}
-          testId="network-management-type"
-        />
-        <DetailItem
-          title="Networking Type"
-          value={getNetworkType(cluster.networkType)}
-          testId="networking-type"
-        />
-      </DetailList>
-    </GridItem>
-    <GridItem md={6}>
-      <DetailList>
-        <DetailItem
-          title="Cluster network CIDR"
-          value={selectClusterNetworkCIDR(cluster)}
-          testId="cluster-network-cidr"
-        />
-        <DetailItem
-          title="Cluster network host prefix"
-          value={selectClusterNetworkHostPrefix(cluster)}
-          testId="host-prefix"
-        />
-        <DetailItem
-          title="Service network CIDR"
-          value={selectServiceNetworkCIDR(cluster)}
-          testId="service-network-cidr"
-        />
-        <DetailItem
-          title="Disk encryption"
-          value={getDiskEncryptionEnabledOnStatus(cluster.diskEncryption?.enableOn)}
-          isHidden={cluster.diskEncryption?.enableOn === 'none'}
-          testId="disk-encryption"
-        />
-        <ClusterFeatureSupportLevelsDetailItem cluster={cluster} />
-      </DetailList>
-    </GridItem>
-  </>
-);
+          <DetailItem
+            title="Cluster network host prefix (IPv4)"
+            value={selectClusterNetworkHostPrefix(cluster)}
+            testId="host-prefix-ipv4"
+          />
+          {isDualstack ? (
```

Comment:
A minor thing, but if we only render in one clause, I'd use the pattern `isDualstack && render()` instead of `isDualStack ? render() : undefined`

Reply:
I wanted to but I get errors when I try that: 
```
Type 'false | Element' is not assignable to type 'string | number | ReactElement<any, string | JSXElementConstructor<any>> | (ReactElement<any, string | JSXElementConstructor<any>> & string) | undefined'.
Type 'false' is not assignable to type 'string | number | ReactElement<any, string | JSXElementConstructor<any>> | (ReactElement<any, string | JSXElementConstructor<any>> & string) | undefined'.
```
The same construction is used earlier with `externalMode`.

Reply:
No problem, thanks for checking :)